avail/src/main/kotlin/avail/environment/actions/AboutAction.kt

```diff
@@ -6,16 +6,16 @@
  * Redistribution and use in source and binary forms, with or without
  * modification, are permitted provided that the following conditions are met:
  *
- * * Redistributions of source code must retain the above copyright notice, this
```

Comment:
After the previous global copyright fix is done, this one should have been reverted.  If not... it didn't work.  :-)